HGNC_Genomic_Files_Migration/README.md

```diff
@@ -0,0 +1,31 @@
+## HGNC Genomic Data Migration Tool
+
+This tool is intended to migrate the changes in entrez ids and hugo symbols based on HGNC in data files. The tool,
+1. Replaces the outdated entrez ids/hugo symbols to new entrez ids/hugo symbols. The outdated to new entrez id/hugo symbols mappings are pre-defined based on our analysis and is included in the script folder.
```

Comment:
"The ... mappings are actively maintained/updated..".

Reply:
Done